src/main/java/by/epam/nickgrudnitsky/model/output/FilePrinter.java

```diff
@@ -0,0 +1,27 @@
+package by.epam.nickgrudnitsky.model.output;
+
+import java.io.*;
+
+public class FilePrinter implements Printer {
+    private File file;
+
+    public FilePrinter(String fileName) {
+        this.file = new File(fileName);
+    }
+
+    @Override
+    public void print(String message)  {
+        try {
+            if (!file.exists()){
+                file.createNewFile();
```

Comment:
looks a bit weird. `createNewFile()` have a check inside that file exists or not before creating